### DISCUSSION TITLE
web3modal isn't opening in popup

### DISCUSSION THREAD

**Discussion by Kam125** [WalletConnect/walletconnect-monorepo#3720] [Developer Support]
Hello guys, I'm using the latest documentation for Wallet connect, the issue is, when I click on the connect wallet button, instead of opening web3modal it scrolls the website to the end of the page and shows the wallet connect UI there and does not in the popup or modal. Is this issue on my end, or you are facing it as well?

**Comment by glitch-txs** [comment]
Hi, we haven't faced this issue, do you have an example we can run and reproduce?

**Comment by Kam125** [comment]
Just follow the docs and install the latest @web3modal/wagmi package.

**Reply by glitch-txs** [reply]
We have this running in labs https://lab.web3modal.com/

**Reply by Kam125** [reply]
Please check it here https://pretiumpresale.pages.dev/presale

**Reply by glitch-txs** [reply]
Is your code public? Could you share any public repo?
